From 91ea01964f19d04a1418ab73733bc9655bf667a8 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Tue, 23 Aug 2011 00:51:52 +0800
Subject: [PATCH 154/249] ENGR00155172 vpu: Fix android linux-lib build error

Fix android vpu lib build error due to ENGR00153838 commit.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 vpu/Android.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/vpu/Android.mk b/vpu/Android.mk
index 177b09e..2a32504 100755
--- a/vpu/Android.mk
+++ b/vpu/Android.mk
@@ -7,7 +7,8 @@ include $(CLEAR_VARS)
 LOCAL_SRC_FILES := \
 	vpu_io.c \
 	vpu_util.c \
-	vpu_lib.c
+	vpu_lib.c \
+	vpu_gdi.c
 LOCAL_CFLAGS += -DBUILD_FOR_ANDROID -D$(BOARD_SOC_TYPE)
 LOCAL_C_INCLUDES += $(LOCAL_PATH)
 LOCAL_MODULE := libvpu
-- 
1.8.0

